src/Admin/TimelineProfileAdmin.php

```diff
@@ -70,4 +70,14 @@ protected function configureListFields(ListMapper $listMapper)
             ])
         ;
     }
+
+    public function getExportFields()
+    {
+        return [
+            'ID' => 'id',
+            'Titre (FR,EN)' => 'exportTitles',
```

Comment:
missing white space here ? `(FR,EN)` > `(FR ,EN)`

Reply:
ok go for `(FR, EN)` 👌